api/package.json

```diff
@@ -4,7 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node src/index.js"
+    "start": "npm run start-db && node src/index.js",
+    "start-db": "brew services start mongodb-community@4.4",
```

Comment:
this will run the mongodb service in the background and use the settings provided in the configuration file at `/usr/local/etc/mongod.conf`

Reply:
this looks completely sensible to me!